src/Components/EmailNotificationForm/EmailNotificationForm.css

```diff
@@ -0,0 +1,73 @@
+.label-input-container, .preferred-lang-p, .form-header {
+  font-size: 20px;
+  color: #2b2d42;
+  display: block;
+}
+
+.state-name-select {
+  font-size: 1.1rem;
+}
+
+em {
+  color: #fd3535;
+}
+
+.input {
+  display: block;
+  width: 177px;
+}
+
+.submit-button {
+  background-color: #fd3535;
+  font-size: 20px;
+  border-radius: 8px;
+  font-weight: bold;
+  width: 150px;
+}
+
+.submit-button:hover {
+  color: white;
+}
+
+.form-header {
+  font-family: 'DM Serif Display', serif;
+  font-size: 28px;
+  background-color: #1990ff;
+  text-align: center;
+}
+
+.form-container {
+  border-radius: 8px;
+  overflow: hidden;
+  box-shadow: 3px 1px 6px 5px rgb(0 0 0 / 20%);
```

Comment:
What does the `/ 20%` mean?  Is that the same as writing `rgba(50 70 30 0.2)` <- I have zero clue what color that is, btw 😊

Reply:
This is for the box-shadow property. From MDN (https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow):
/* offset-x | offset-y | blur-radius | spread-radius | color */, so the rgb(0 0 0 / 0.2) are the colors for each and 0.2 is the percentage of the spread-radius.